keyboards/kiserdesigns/madeline/keymaps/vial/rules.mk

```
@@ -0,0 +1,7 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+COMBO_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+VIAL_ENCODERS_ENABLE = no
+ENCODER_MAP_ENABLE = yes
+VIAL_INSECURE = no
```

```suggestion
ENCODER_MAP_ENABLE = yes
```
The rest are on by default or no longer needed.